### ISSUE TITLE
`@parcel/transformer-webextension` Manifest v3, oauth2 error

### ISSUE THREAD

**Issue by itslooklike** [parcel-bundler/parcel#8017]
# 🐛 bug report

## 🎛 Configuration (.babelrc, package.json, cli command)

```js
{
  "extends": "@parcel/config-webextension"
}
```

## 🤔 Expected Behavior

Success build

## 😯 Current Behavior

give error on `oauth2` field in `manifest.json`

<img width="1044" alt="image" src="https://user-images.githubusercontent.com/17547664/165814533-03c0eea7-83d1-4c9d-9399-047de7962697.png">

## 💁 Possible Solution

add support for field

## 🔦 Context

https://developer.chrome.com/docs/apps/app_identity/

## 💻 Code Sample

```
{
  "manifest_version": 3,
  "name": "Sample Web Extension",
  "version": "0.0.1",
  "permissions": ["tabs", "storage", "identity", "identity.email"],
  "background": {
    "service_worker": "apps/background.ts",
    "type": "module"
  },
  "action": {
    "default_popup": "apps/popup/index.html"
  },
  "chrome_url_overrides": {
    "newtab": "apps/new-tab/index.html"
  },
  "oauth2": {
    "client_id": "__CLIENT_ID_HERE__",
    "scopes": [
      "profile",
      "email",
      "https://www.googleapis.com/auth/userinfo.email",
      "https://www.googleapis.com/auth/userinfo.profile"
    ]
  }
}

```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |2.5.0
| @parcel/config-webextension           |2.5.0
| Node             | 17.9
| npm/Yarn         | yarn 1.22.18
| Operating System | macos 12.3.1

**Comment by mischnic** [comment]
That would have to be added here: https://github.com/parcel-bundler/parcel/blob/50893a7ec2b1490e6d0a2f5e1f9ab6a8fc57a113/packages/transformers/webextension/src/schema.js#L440

**Comment by 101arrowz** [comment]
Resolved by #8037.
